remote

```diff
@@ -1,12 +1,5 @@
 #!/bin/bash
-# Copy to /usr/local/bin/rnsorros
 
-# Either set these here, or in your ~/.bashrc or equivalent.
-# Changing the $INSTANCE_NAME env var will allow you to send commands to 
-
-INSTANCE_NAME=datascience_base
-REMOTE_DATALABS_PATH=/data/datalabs
```

Comment:
why are also removing those?

Reply:
I think it makes sense that they all get set in the local env, as they are not fixed.